dev_guide/builds.adoc

```diff
@@ -84,6 +84,33 @@ The `*ImageChange*` trigger creates a new build each time the Docker image speci
 <4> The `*strategy*` section describes the build strategy used to execute the build. You can specify `*Source*`, `*Docker*` and `*Custom*` strategies here. This specification uses the `*ruby-20-centos7*` Docker image that Source-To-Image will use for the application build.
 <5> After the Docker image is successfully built, it will be pushed into the repository described in the `*output*` section.
 
+== Defining an incremental buildConfig
```

Comment:
I was thinking of my paragraph as being the continuation of the previous one, where buildConfig is used.